R/check-cran.sh

```diff
@@ -20,14 +20,14 @@
 set -o pipefail
 set -e
 
-FWDIR="$(cd `dirname "${BASH_SOURCE[0]}"`; pwd)"
-pushd $FWDIR > /dev/null
+FWDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)"
```

Comment:
you can use like this:
FWDIR="$(cd "`dirname "${BASH_SOURCE[0]}"`"; pwd)" 

But i used the "$( )", it also works. You can test this. 
Thank you.